addons/mail/static/src/core/common/link_preview_confirm_delete.js

```diff
@@ -19,12 +19,23 @@ export class LinkPreviewConfirmDelete extends Component {
 
     setup() {
         this.rpc = useService("rpc");
+        this.store = useService("mail.store");
     }
 
     onClickOk() {
         this.rpc(
             "/mail/link_preview/delete",
-            { link_preview_id: this.props.linkPreview.id },
+            { link_preview_ids: [this.props.linkPreview.id] },
+            { shadow: true }
```

Comment:
Note that `shadow: true` works for legacy, but this param was renamed `silent: true`.
It seems that was the only occurrence in `mail/` with this typo, could be worth changing that in this diff!